web/modules/custom/ai_screening/src/Helper/UserHelper.php

```diff
@@ -48,12 +73,23 @@ public function setDefaultLanguage(UserInterface $user): UserInterface {
     return $user;
   }
 
+  /**
+   * Event handler for altering the username on display.
+   */
+  public function alterUserName(UserFormatNameAlterEvent $event): void {
```

Comment:
Are we sure that we actually want to always alter the user name (for display)? This will also affect the form when an administrator edits a user.

Maybe being explicit about what we want to display in our templates (e.g. `user.name` or `'user.field_name`) is a better approach.

Reply:
Well. In my opinion we never really want to display the drupal username. I don't see the issue with edit user form?